cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/RunIds.java

```diff
@@ -77,12 +78,13 @@ public static RunId generate(long timeMillis) {
   }
 
   /**
-   * @return time in milliseconds from the UUID if it is a time-based UUID, -1 otherwise.
+   * @return time from the UUID if it is a time-based UUID, -1 otherwise.
    */
-  public static long getTimeMillis(RunId runId) {
```

Comment:
Why remove this method? you shouldn't. I was suggesting to add the `getTime`, not to replace this one.